pkg/apiserver/apiserver.go

```diff
@@ -258,7 +258,13 @@ func write(statusCode int, apiVersion string, codec runtime.Codec, object runtim
 		w.WriteHeader(statusCode)
 		writer := w.(io.Writer)
 		if flush {
-			writer = flushwriter.Wrap(w)
+			if _, ok := w.(*hijackTimeoutWriter); ok {
```

Comment:
@avalamp Thanks for advise, I have submitted new code. It could avoid type conversions.